Build lldb-tblgen during cross builds even if it was provided as host executable.
--- a/lldb/utils/TableGen/CMakeLists.txt
+++ b/lldb/utils/TableGen/CMakeLists.txt
@@ -1,8 +1,8 @@
 # tablegen targets get exported via llvm for LLVMConfig.cmake. So standalone
 # builds of lldb can potentially import this via LLVMConfig and also attempt to
 # build it in tree. So only build it if it doesn't exist.
-if (NOT DEFINED LLDB_TABLEGEN_EXE)
-  if (TARGET lldb-tblgen)
+if (TRUE)
+  if (FALSE)
     set(LLDB_TABLEGEN_EXE $<TARGET_FILE:lldb-tblgen> CACHE STRING "")
   else()
     set(LLVM_LINK_COMPONENTS Support)
